src/main/java/apoc/schema/Schemas.java

```diff
@@ -36,6 +36,23 @@
     @Context
     public KernelTransaction tx;
 
+    @Procedure(value = "apoc.schema.recreate", mode = Mode.SCHEMA)
+    @Description("apoc.schema.recreate() - recreates indexes and constraints")
+    public Stream<AssertSchemaResult> schemaRecreate(@Name(value = "config", defaultValue = "{}") Map<String, Object> config) throws ExecutionException, InterruptedException, TimeoutException {
+        long allIndexesTimeout = (long) config.getOrDefault("retrieveIndexesTimeout", 10000L);
```

Comment:
the timeouts are bit low
esp. on large graphs those can be hours, so I would increase them massively (which doesn't hurt)

Reply:
where do the current numbers come from?

Reply:
I don't think the operations are synchronous though. The actual command to drop or create should happen straight away I think? And then in the background there would be work going on in an async fashion

Reply:
right for indexes, for constraints it blocks

Reply:
sorry I was referring to db.awaitIndexes() timeouts

Reply:
so for this it's fine as it is?